datasets/youtube_caption_corrections/README.md

```diff
@@ -0,0 +1,169 @@
+0: No difference
+1: Case based difference, e.g. `hello` vs `Hello`
+2: Punctuation difference, e.g. `hello` vs `hello`
+3: Case and punctuation difference, e.g. `hello` vs `Hello,`
+4: Word difference with same stem, e.g. `thank` vs `thanked`
+5: Digit difference, e.g. `2` vs `two`
+6: Intra-word punctuation difference, e.g. `autogenerated` vs `auto-generated`
+7: Unknown type of difference, e.g. `laughter` vs `draft`
+8: Reserved for unspecified difference
+
+{
+    'video_titles': '_QUEXsHfsA0', 
+    'default_seq': ['you', 'see', "it's", 'a', 'laughter', 'but', 'by', 'the', 'time', 'you', 'see', 'this', 'it', "won't", 'be', 'so', 'we', 'have', 'a', 'big']
+    'correction_seq':  ['', 'see,', '', '', 'draft,', '', '', '', '', '', 'read', 'this,', '', '', 'be.', 'So', '', '', '', '']
+    'diff_type': [0, 2, 0, 0, 7, 0, 0, 0, 0, 0, 7, 2, 0, 0, 2, 1, 0, 0, 0, 0]
+}
+
+### Data Fields
+
+- 'video_ids': Unique ID used by YouTube for each video. Can paste into `https://www.youtube.com/watch?v=<{video_ids}` to see video
+- 'default_seq': Tokenized auto-generated YouTube captions for the video
+- 'correction_seq':  Tokenized manually-corrected YouTube captions only at those locations, where there is a difference between the auto-generated and manually-corrected captions
+- 'diff_type': A value greater than `0` at every token where there is a difference between the auto-generated and manually-corrected captions
+
+### Data Splits
+
+No data splits
+
+## Dataset Creation
+
+### Curation Rationale
+
+It was created after viewing errors in auto-generated captions at a recent virtual conference, with the hope that there could be some way to help correct those errors.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+All captions are requested via `googleapiclient` and `youtube_transcript_api` at the `channel_id` and language granularity, using scripts written at https://github.com/2dot71mily/youtube_captions_corrections.
+
+The captions are tokenized on spaces and the manually-corrected sequence has here been reduced to only include differences between it and the auto-generated sequence.
+
+#### Who are the source language producers?
+
+Auto-generated scripts are from YouTube and the manually-corrected scripts are from creators, and any support they may have (e.g. community or software support)
+
+### Annotations
+
+#### Annotation process
+
+Scripts at repo, https://github.com/2dot71mily/youtube_captions_corrections take a diff of the two captions and use this to create annotations.
+
+#### Who are the annotators?
+
+YouTube creators, and any support they may have (e.g. community or software support)
+
+### Personal and Sensitive Information
+
+All content publicly available on YouTube
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
```

Comment:
```suggestion
[More Information Needed]

### Discussion of Biases

[More Information Needed]

### Other Known Limitations

[More Information Needed]
```
we use [More Information Needed] in general

Reply:
Thanks, will do